framework/keyring-interface.md

```diff
@@ -93,7 +97,7 @@ and SHOULD be equal to a UTF-8 encoding of the key name.
 
 ## Supported Keyrings
 
-- [AWS KMS Keyring](kms-keyring.md)
+- [AWS KMS Keyrings](./aws-kms/)
```

Comment:
I think that this is "fine".
The idea was to link to the "real" keyring.
There are like 4 or so now...
This link will go to "all the KMS stuff" as opposed to the specific keyrings...

Reply:
Indeed, this is the smallest change that fixes the links to point to a "more" correct location. 
If you're keen, I can make a larger change to organize to something like:

```
framework/keyrings/keyring-interface.md
framework/keyrings/multi-keyring.md
...
framework/keyrings/aws-kms/aws-kms-keyring.md
framework/keyrings/aws-kms/aws-kms-mrk-discovery-keyring.md
...
```

Reply:
Let's make a full list.
At this point there are quite a few
and I can see us missing one...
It would be good to have a nice single list _somewhere_.

I don't think we need to keep this list _everywhere_
So maybe we just update the list here, and leave the rest of the links the same